### PR TITLE
Update introduction.md

### PR DESCRIPTION
**Pull request by Doni108doni108**

### CODE REVIEW

File: src/guide/introduction.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-footer: false
+footer: true
```
Comment (NataliaTepluhina) [inline src/guide/introduction.md:2]: **question**: could you please provide a little bit more context on why this change has been made?

Comment (Doni108doni108) [reply src/guide/introduction.md:2]: - [ ]

**Comment by NataliaTepluhina** [comment]
Closing the MR as there has been no response from the author for two weeks.

**Review by Doni108doni108 — COMMENTED** [review]
U
